examples/jsm/math/Random.js

```diff
@@ -0,0 +1,18 @@
+function onUnitSphere( vec ) {
+
+	// Derived from https://mathworld.wolfram.com/SpherePointPicking.html
+
+	const u = ( Math.random() - 0.5 ) * 2;
+	const t = Math.random() * Math.PI * 2;
+	const f = Math.sqrt( 1 - u ** 2 );
+
+	vec.x = f * Math.cos( t );
+	vec.y = f * Math.sin( t );
+	vec.z = u;
+
+	return vec;
+
+}
+
+
+export { onUnitSphere };
```

Comment:
One issue I noticed with this export approach is that `npm run build-examples` will produce an `examples/js/math/Random.js` file that puts the function directly on `THREE.onUnitSphere`, which seems wrong. I suppose our choices are to either export a Random class with static function, add Random to the ignore list in rollup.examples.config.js, or write the js version manually.

Reply:
I guess this can be fixed by naming the file `RandomUtils.js`. The example build script will then perform the correct conversion.

Reply:
Ah, nice. That works perfectly. Added a commit to rename to RandomUtils.